velox/functions/prestosql/window/tests/SimpleAggregatesTest.cpp

```diff
@@ -28,19 +28,21 @@ class SimpleAggregatesTest : public WindowTestBase {
 
   RowVectorPtr makeBasicVectors(vector_size_t size) {
     return makeRowVector({
-        makeFlatVector<int32_t>(
-            size, [](auto row) -> int32_t { return row % 10; }),
-        makeFlatVector<int32_t>(
-            size, [](auto row) -> int32_t { return row % 7; }),
-        makeFlatVector<int32_t>(size, [](auto row) -> int32_t { return row; }),
+        makeFlatVector<int64_t>(size, [](auto row) { return row % 10; }),
+        makeFlatVector<int64_t>(size, [](auto row) { return row % 7; }),
+        // k in k PRECEDING / FOLLOWING frame bounds, represented by c2 in
```

Comment:
Thanks for the suggestion, c2 and c1 are now documented since it should always have a value greater than 0.
The type was changed for c2 to 64-bit as suggested by @aditi-pandit : https://github.com/facebookincubator/velox/pull/3325#discussion_r1083080675
It was changed to int64 for the rest of the columns for consistency. Please let me know if it should be reverted to int32.

Reply:
> The type was changed for c2 to 64-bit as suggested by @aditi-pandit : https://github.com/facebookincubator/velox/pull/3325#discussion_r1083080675

If this is for the existing code, would you extract this change into a separate PR and provide reasons for the change in the PR description?

Reply:
Reverted the change to int64 for column c0 which is not used as a frame bound, this change is needed for c1 though to read the frame offset correctly.